libraries/chain/database.cpp

```diff
@@ -1490,21 +1457,37 @@ void database::clear_null_account_balance()
          g.pending_rewarded_vesting_shares -= null_account.reward_vesting_balance;
          g.pending_rewarded_vesting_steem -= null_account.reward_vesting_steem;
       });
-
-      modify( null_account, [&]( account_object& a )
-      {
-         a.reward_vesting_steem.amount = 0;
-         a.reward_vesting_balance.amount = 0;
-      });
    }
 
+   // We simply set all account balances to zero here because of the newly added
+   // assertions in the `adjust_*` family of functions that prevents negative deltas
+   // to the null account.
+   modify( null_account, [&]( account_object& a )
```

Comment:
Put this in a conditional `if( total_steem.amount > 0 || total_sbd.amount > 0 )` so that we do not do the modifications every block.